cmd/translate/main_grpc_integration_test.go

```diff
@@ -35,6 +36,8 @@ var (
 	client translatev1.TranslateServiceClient
 )
 
+const name = "translate.server.integration_test"
```

Comment:
I suggest declaring package level tracer and using it everywhere, reducing code boilerplate code.

The `name` in `otel.Tracer(name)` is stored in the attribute [otel.library.name](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/mapping-to-non-otlp.md#instrumentationscope) - deprecated and alias to `otel.scope.name`. [More about Instrumentation Scope ](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope)

Based on the Instrumentation Scope description, I suggest the name `go.expect.digital/translate/cmd/translate`.

Reply:
two things:
1. both must be defined otel.library.name + otel.scope.name
 > The following deprecated aliases MUST also be reported with exact same values for backward compatibility reasons:
2. Should we wait for OpenTelemetry to change this behavior in the Go package when creating a new tracer? While we can manually add a new attribute 'otel.scope.name' to traces that we create, all other instrumentation (such as gRPC, HTTP, and SQL) will still only have otel.library.name. That means inconsitency

Reply:
We do nothing about it - We don't add manually `otel.scope.name`, it is the responsibility of the OTEL go package. They are gonna do it when the time comes.

Reply:
My understanding is, both are required to not break any behavior by collectors and GUIs. Otherwise, they are only recommended attributes.